app/experimenter/nimbus-ui/src/components/PageHome/DirectoryTable/index.tsx

```diff
@@ -340,12 +340,15 @@ const DirectoryTable: React.FunctionComponent<DirectoryTableProps> = ({
   }
 
   return (
-    <div
-      className="directory-table pb-2 mt-4"
-      style={{ width: "auto", maxWidth: "100%" }}
-    >
+    <>
```

Comment:
Can we take this out?

Reply:
interesting, actually no 😆 , this is a fragment, as the div is removed we need to include this

Reply:
🤔 🤔 🤔